src/Model/TransportModel/tsp1ssm1.f90

```diff
@@ -353,27 +367,27 @@ subroutine ssm_term(this, ipackage, ientry, rrate, rhsval, hcofval, &
     ! -- set requested values
     if (present(hcofval)) hcofval = hcoftmp
     if (present(rhsval)) rhsval = rhstmp
-    if (present(rrate)) rrate = hcoftmp * ctmp - rhstmp
+    if (present(rrate)) rrate = (hcoftmp * ctmp - rhstmp)
```

Comment:
Is there supposed to be a change here?

Reply:
Not sure why this crept in, has been removed with 609209d